modules/ossm-cr-example.adoc

```diff
@@ -183,7 +183,7 @@ The following table lists the specifications for the `ServiceMeshControlPlane` r
 |Name |Description |Configurable parameters
 
 |addons
-|Addons is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
+|Addon is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
```

Comment:
Shouldn't this be `Addons`? I read this as referring to the field name.... FieldName is used to configure...